src/reducers/adminReducer.js

```diff
@@ -76,6 +81,20 @@ const updatePriceRule = (state, payload, action) => {
     }
 };
 
+const updateUnpaid = (state, payload, action) => {
+    let {response} = payload
+    let {data} = response;
+    let {Unpaid} = state;
+    switch(action) {
+        case "GET":{
+            
+            // Unpaid = data;
+            console.log(Unpaid)
+            return Unpaid;
```

Comment:
This will override all of the admin redux state when it returns! A new copy of the *entire* state needs to be returned from the reducer, not just the part you modifier. A fix would be like so:

    return {
        ...state,
        Unpaid,
    };

This copies over all other things from the previous state, replacing only `Unpaid`.